src/pages/Payment.vue

```diff
@@ -84,11 +84,11 @@ export default {
         onRequestPayment() {
             Store.pendRecVouchers(
                 // on Success, route to /account
-                function() {
+                function(response) {
```

Comment:
I think so, will JS throw a fit if it's passed a variable in a function when it's not expecting it?